vizro-core/docs/pages/user-guides/figure.md

```diff
@@ -105,3 +105,238 @@ To add a `Figure` to your page, do the following:
+                        title="KPI with icon",
+                        icon="shopping_cart",
+                    )
+                ),
+                vm.Figure(
+                    figure=kpi_card_reference(
+                        data_frame=df,
+                        value_column="Reference",
+                        reference_column="Actual",
+                        title="KPI with reference (neg)",
+                    )
+                ),
+                vm.Figure(
+                    figure=kpi_card_reference(
+                        data_frame=df,
+                        value_column="Actual",
+                        reference_column="Reference",
+                        title="KPI with reference (pos)",
+                    )
+                ),
+                vm.Figure(
+                    figure=kpi_card_reference(
+                        data_frame=df,
+                        value_column="Reference",
+                        reference_column="Actual",
+                        title="KPI with reference and custom formatting",
+                        value_format="{value:.2f}$",
+                        reference_format="{delta:.2f}$ vs. last year ({reference:.2f}$)",
+                    )
+                ),
+                vm.Figure(
+                    figure=kpi_card_reference(
+                        data_frame=df,
+                        value_column="Actual",
+                        reference_column="Reference",
+                        value_format="${value:.2f}",
+                        title="KPI with reference and icon",
+                        icon="shopping_cart",
+                    ),
+                ),
+            ],
+            controls=[vm.Filter(column="Category")],
+        )
+
+
+        dashboard = vm.Dashboard(pages=[page])
+        Vizro().build(dashboard).run()
+        ```
+    === "app.yaml"
+        ```yaml
+        # Still requires a .py to add data to the data manager and parse YAML configuration
+        # See from_yaml example
+        pages:
+          - components:
+              - figure:
+                  _target_: kpi_card
+                  data_frame: tips
+                  value_column: tip
+                  value_format: ${value:.2f}
+                  icon: shopping_cart
+                  title: KPI Card I
+                type: figure
+            controls:
+              - column: day
+                type: filter
+                selector:
+                  type: radio_items
+            layout:
+              grid:
+                [
+                  [0, -1, -1, -1],
+                  [-1, -1, -1, -1],
+                  [-1, -1, -1, -1],
+                  [-1, -1, -1, -1],
+                  [-1, -1, -1, -1],
+                ]
+            title: KPI Indicators
+        ```
+    === "Result"
+        [![KPICards]][KPICards]
+
+    [KPICards]: ../../assets/user_guides/figure/kpi_cards.png
+
+
+#### Custom KPI Card function
```

Comment:
I was planning to add a custom kpi card function with the reverse color logic here @antonymilne. Do you think this is useful? Or shall we kill this section alltogether?

Reply:
Answered elsewhere already, but in case it's not clear: let's remove this from here and just do on the custom figure page instead.